2016/ideas-list-swc.md

```diff
@@ -1 +1,115 @@
 # Software Carpentry
+
+## Produce a SurveyMonkey responses analyzer application
```

Comment:
This may not be a good title ("analyzer"? Should be something with visual representation). @gvwilson can you perhaps suggest something?